stable/minio/templates/statefulset.yaml

```diff
@@ -62,6 +62,15 @@ spec:
                 secretKeyRef:
                   name: {{ template "minio.fullname" . }}
                   key: secretkey
+          livenessProbe:
+            tcpSocket:
+              port: service
+            initialDelaySeconds: 5
```

Comment:
would it make sense to make initialDelaySeconds and timeoutSeconds configurable (and how many retries) - or are these bound to always be fast?

Reply:
@davidkarlsen that makes sense to have configurable with these default values and if the user wants to change will be easier. I will add that

thanks for the review

Reply:
@davidkarlsen PTAL